modules/nf-core/custom/dumpsoftwareversions/environment.yml

```diff
@@ -1,7 +1,7 @@
-name: custom_dumpsoftwareversions
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
 channels:
   - conda-forge
   - bioconda
-  - defaults
 dependencies:
-  - bioconda::multiqc=1.19
+  - bioconda::multiqc=1.27
```

Comment:
I suspect this a nf-core-specific thing (because it uses the multiqc container), but do you know if dumpsoftwareversions actually depends on multiqc or is it a weird legacy thing?

Reply:
I'm not sure. I suspect it doesn't and this is more that multiqc is integrated into the standard nf-core template. I did not modify any of the nf-core modules code, just updated.